kpi/models/object_permission.py

```diff
@@ -830,7 +832,7 @@ def assign_perm(self, user_obj, perm, deny=False, defer_recalc=False,
         for implied_perm in implied_perms:
             self.assign_perm(
                 user_obj, implied_perm, deny=deny, defer_recalc=True)
-        # We might have been called by ourself to assign a related
+        # We might have been called by ourselves to assign a related
```

Comment:
Well, yes, this was weirdly written, but: https://www.merriam-webster.com/dictionary/ourself
Then, from there, you can waste tons of time reading people argue about the use of "royal we" in technical and scientific writing :zany_face:

Reply:
My bad! I thought this comment was created by one of my fellow countrymen and I thought it was a mistake. 
I'll revert it back.

Reply:
I think there are just as many (or more) arguments for `ourself` being wrong here as right :joy: